event_routing_backends/models.py

```diff
@@ -40,6 +42,50 @@ def get_value_from_dotted_path(dict_obj, dotted_key):
     return result
 
 
+class RouterConfigurationManager(ConfigurationModelManager):
+    """
+    Query manager for ConfigurationModel.
+    """
+
+    def get_routers(self, backend_name):
+        """
+        Bring active routers of a backend.
+
+        A queryset for the active configuration entries only. Only useful if backend_name is passed.
+        This function will return all active routers of a backend.
+        """
+        if backend_name == '':
+            return []
+
+        cache_key = self.cache_key_name(backend_name)
+        cached_response = TieredCache.get_cached_response(cache_key)
+        if cached_response.is_found:
+            return cached_response.value
+
+        current = self.current_set().filter(backend_name=backend_name, enabled=True).order_by('-change_date')
+        cache_key = self.cache_key_name(backend_name)
+        TieredCache.set_all_tiers(cache_key, current, 600)
```

Comment:
for cache timeout we should define a method in helpers.py
```
def backend_cach_ttl():
    return  getattr(settings, 'EVENT_TRACKING_BACKENDS_CACHE_TTL', 600)
```
and use that method to get cache time out

Reply:
Done